presto-testng-services/src/main/java/io/prestosql/testng/services/ReportIllNamedTest.java

```diff
@@ -16,14 +16,24 @@
 import org.testng.IClassListener;
 import org.testng.ITestClass;
 
+import static io.prestosql.testng.services.Listeners.reportListenerFailure;
 import static io.prestosql.testng.services.ReportUnannotatedMethods.isTemptoClass;
-import static java.lang.String.format;
 
 public class ReportIllNamedTest
         implements IClassListener
 {
     @Override
     public void onBeforeClass(ITestClass testClass)
+    {
+        try {
```

Comment:
Can you move wrapping into `try/catch` to `Listeners` class

Reply:
I don't think it buys us anything. The mechanism still needs to be opted in by a listener.
This way is IMO more readable. (exception flow is not really FP friendly)

Reply:
Yeah. The buy in is negligible. Possible we can avoid a bit of repetition from `catch` clause